plugins/org.jkiss.dbeaver.ext.cubrid/plugin.xml

```diff
@@ -57,6 +57,12 @@
                                         </items>
                                     </items>
                                 </folder>
+                                <folder type="org.jkiss.dbeaver.ext.cubrid.model.CubridPartition" label="%tree.partitions.node.name" icon="#partitions" description="%tree.partitions.node.tip">
+                                    <items label="%tree.partitions.node.name" path="partition" property="partitions" icon="#partition">

```

Comment:
Like in the Postgres bundle - it has to be items label="%tree.partition.node.name"
And also register tree.partition.node.name in the plugins/org.jkiss.dbeaver.ext.cubrid/OSGI-INF/l10n/bundle.properties
It all for navigator tree and properties items/commands

Reply:
I have followed the same **label** and register **bundle.properties** as Postgres. 

Do you mean changing from "%tree.**partitions**.node.name" to "%tree.**partition**.node.name"?

Reply:
Correct.